RxSwift/Traits/Single.swift

```diff
@@ -327,4 +327,17 @@ extension PrimitiveSequenceType where TraitType == SingleTrait {
     public func asMaybe() -> Maybe<ElementType> {
         return Maybe(raw: primitiveSequence.source)
     }
+
+    /// Converts `self` to `Completable` trait.
+    ///
+    /// - returns: Completable trait that represents `self`.
+    public func asCompletable() -> Completable {
+        return Completable.create { completable -> Disposable in
```

Comment:
Wouldn't it be better to just do `asObservable().ignoreElements()` ? 

I don't love the idea of subscribing inside a creation-block and using a side effect-ish kind of approach to feeding the completable.

Reply:
I didn't know about `ignoreElements`. It's really better this way!